ember-app/app/components/hb-filter-group.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+var HbFilterGroupComponent = Ember.Component.extend({
+  classNames: ['filter-group'],
+  classNameBindings: ['isCollapsed:collapsed'],
+  isCollapsed: true,
+  click: function(el){
+    if(el.target.className === 'filter-header'){ this.toggleProperty('isCollapsed'); }
```

Comment:
Doesn't using `nodeName` as the selector seem more easily broken? Any future work would more than likely preserve the class structure, even if DOM elements get swapped was my original thought. I could be wrong though.